src/Data/List/Relation/Binary/Disjoint/Setoid/Properties.agda

```diff
@@ -8,49 +8,65 @@
 
 module Data.List.Relation.Binary.Disjoint.Setoid.Properties where
 
+open import Function.Base using (_∘_)
+open import Function.Bundles using (_⇔_; mk⇔)
 open import Data.List.Base
 open import Data.List.Relation.Binary.Disjoint.Setoid
 import Data.List.Relation.Unary.Any as Any
 open import Data.List.Relation.Unary.All as All
 open import Data.List.Relation.Unary.All.Properties using (¬Any⇒All¬)
 open import Data.List.Relation.Unary.Any.Properties using (++⁻)
-open import Data.Product.Base using (_,_)
+import Data.List.Membership.Setoid.Properties as Mem
+open import Data.Product.Base as Product using (_,_)
 open import Data.Sum.Base using (inj₁; inj₂)
 open import Relation.Binary.Bundles using (Setoid)
-open import Relation.Binary.Definitions using (Symmetric)
+open import Relation.Binary.Definitions using (Symmetric; DecidableEquality)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_)
 open import Relation.Nullary.Negation using (¬_)
 
-------------------------------------------------------------------------
--- Relational properties
-------------------------------------------------------------------------
-
 module _ {c ℓ} (S : Setoid c ℓ) where
 
-  sym : Symmetric (Disjoint S)
-  sym xs#ys (v∈ys , v∈xs) = xs#ys (v∈xs , v∈ys)
+  open Setoid S using (_≈_; reflexive) renaming (Carrier to A)
 
-------------------------------------------------------------------------
--- Relationship with other predicates
-------------------------------------------------------------------------
+  ------------------------------------------------------------------------
+  -- Relational properties
+  ------------------------------------------------------------------------
 
-module _ {c ℓ} (S : Setoid c ℓ) where
+  sym : Symmetric (Disjoint S)
+  sym xs#ys (v∈ys , v∈xs) = xs#ys (v∈xs , v∈ys)
 
-  open Setoid S
+  ------------------------------------------------------------------------
+  -- Relationship with other predicates
+  ------------------------------------------------------------------------
 
   Disjoint⇒AllAll : ∀ {xs ys} → Disjoint S xs ys →
                     All (λ x → All (λ y → ¬ x ≈ y) ys) xs
```

Comment:
`Setoid` now defines `_≉_` so could (more) simply write
```suggestion
                    All (λ x → All (x ≉_) ys) xs
```
or even in the spirit of #2478 use `All (_∉ ys) xs` instead?

Reply:
Sorry! Out of scope for this PR!